WireUI/Sources/WireDesign/Appearance/SidebarViewDesign.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+public struct SidebarViewDesign {
+
+    public let backgroundColor = ColorTheme.Backgrounds.background
+    public let accountInfoViewDisplayNameColor = ColorTheme.Backgrounds.onSurface
+    public let accountInfoViewUsernameColor = ColorTheme.Base.secondaryText
+    public let menuItemTitleForegroundColor = ColorTheme.Backgrounds.onBackground
+    public let menuItemLinkIconForegroundColor = ColorTheme.Base.secondaryText
+    public let menuItemIsSelectedTitleForegroundColor = ColorTheme.Base.onPrimary
+
+    public init() {}
+}
```

Comment:
I really like this pattern on how we define colors for a component!

Reply:
To be honest, I'm not sure yet. :D